tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@react-native/typescript-config/tsconfig.json",
+  "compilerOptions": {
+    "types": ["react-native"],
+    "baseUrl": "./",
```

Comment:
`baseUrl` is used for aliasing by setting a base path (e.g. can import it like `src/dir/filename` instead of `./dir/filename`) - bob doesn’t support aliasing so auto imports etc. can easily break this. i also recommend including `rootDir: '.'` for consistent output paths regardless of folder structure where rootDir is determined by `tsc`

Reply:
good catch!